cms/templates/js/license-selector.underscore

```diff
@@ -0,0 +1,106 @@
+                <% if (optionInfo.type == "boolean") { %>
+                    <% var optionSelected = model.options[optionKey]; %>
+                    <% var optionDisabled = optionInfo.disabled ||
+                            (optionInfo.conflictsWith && _.any(optionInfo.conflictsWith, function(key) {return model.options[key];}));
+                    %>
+                    <li data-option="<%- optionKey %>"
+                            class="action-item license-button
+                                   <% if (optionSelected) { print("is-selected"); } %>
+                                   <% if (optionDisabled) { print("is-disabled"); } else { print("is-clickable"); } %>"
+                    >
+                        <i class="fa fa-fw
+                             <% if(optionSelected) { print("fa-check-square-o"); } else { print("fa-square-o"); } %>
+                             <% if(optionDisabled) { print("is-disabled"); } %>
+                             "></i>
+                        <h4 class="option-name"><%- optionInfo.name %></h4>
+                        <div class="explanation"><%- optionInfo.help %></div>
+                    </li>
+                <% } // could implement other types here %>
+            <% }) %>
+        </ul>
+    </div>
+<% } %>
+
+<% if (showPreview) { %>
+    <div class="license-preview-wrapper">
+        <label class="label setting-label" for="license-preview">
+            <%= gettext("License Display") %>
+        </label>
+        <p class="tip">
+            <%= gettext("The following message will be displayed at the bottom of the courseware pages within your course.") %>
+        </p>
+        <div id="license-preview">
+        <% // keep this synchronized with the contents of common/templates/license.html %>
+        <% if (model.type === "all-rights-reserved") { %>
+            © <span class="license-text"><%= gettext("All Rights Reserved") %></span>
+        <% } else if (model.type === "creative-commons") {
+             var possible = ["by", "nc", "nd", "sa"];
+             var enabled = _.filter(possible, function(option) {
+                 return model.options[option] === true || model.options[option.toUpperCase()] === true;
+             });
+             var version = model.options.ver || "4.0";
+             if (_.isEmpty(enabled)) {
+                    enabled = ["zero"];
+                    version = model.options.ver || "1.0";
+             }
+        %>
+            <a rel="license" href="//creativecommons.org/licenses/<%- enabled.join("-") %>/<%- version %>/">
+            <% if (previewButton) { %>
+                <img src="https://licensebuttons.net/l/<%- enabled.join("-") %>/<%- version %>/<%- typeof buttonSize == "string" ? buttonSize : "88x31" %>.png"
+                    alt="<%- typeof licenseString == "string" ? licenseString : "" %>"
+                    />
+            <% } else { %>
+                <i class="icon-cc"></i>
```

Comment:
until we change over to using SVG for icons, we need to put [```aria-hidden="true"```](https://edx.readthedocs.org/projects/edx-developer-guide/en/latest/accessibility.html#some-cautions-for-using-wai-aria) on the ```<i>``` element.  Apply to icon below as well.